Source/WebCore/Modules/fetch/FetchBodyOwner.cpp

```diff
@@ -308,7 +309,7 @@ FetchBodyOwner::BlobLoader::BlobLoader(FetchBodyOwner& owner)
 
 void FetchBodyOwner::BlobLoader::didReceiveResponse(const ResourceResponse& response)
 {
-    if (response.httpStatusCode() != 200)
+    if (response.httpStatusCode() != httpStatusOk)
```

Comment:
FWIW, I personally would find this less readable - I know what status 200 is, but not what `Ok` is.

Reply:
Maybe we should just use the status code in the constant name? e.g. "httpStatus200" or "httpStatusCode200"?

Reply:
https://www.youtube.com/watch?v=vqgSO8_cRio